docs/interchain-staking.md

```diff
@@ -0,0 +1,178 @@
+
+In order for the parent chain to relay validator sets and voting powers to the child chain, it needs to be able to distinguish between validator sets relevant to different networks. Not all validators within a parent chain staking module will have opted-in to being part of each child chain set. In order to support Interchain Staking, the parent chain will need to be extended to differentiate between sets of validators and their voting powers with respect to various chains as designated within the Chain Registry.
+
+To provide the necessary functionality on the parent chain, a wrapper module may need to be implemented that will collate staking module validators with regard to their inclusion in sets stored within the Chain Registry. A module like this will need to import both the Chain Registry Keeper and the Staking Keeper in order to make chain specific staking queries. These queries will be requested periodically by the Cross Chain Validation module and relayed to the child chain via IBC. This module can be referred to as the "xStaking Module".
+
+
+### Validator set limits
+
+The current Cosmos Hub has a limit of 125 validators. This limit is imposed on validators who are interested in producing blocks as part of a validator set of the Cosmos Hub itself. This limits the number of eligible validators for child chains to the same top 125 Cosmos Hub participants. However, just because a validator doesn't have enough staked ATOM to be eligible to validate on the Cosmos Hub, doesn't mean that they shouldn't qualify to validate on another child chain.
+
+Interchain Staking should increase the diversity of the validator ecosystem by lowering the barrier to running a profitable validator business. This will go far in creating a healthy ecosystem of diverse validators that will result in anti-fragile and robustly operated networks. In order to make it possible for the top 125 validators to remain eligible as block producers for the Cosmos Hub while increasing the number of eligible validators for child chains, the Staking Module needs to stop forcing validators to undelegate when they leave the top set of 125 validators. This will result in a longer list of validators with ATOM delegations that are not participating in block production on the parent chain (Cosmos Hub). These extra validators will however be eligible to produce blocks on child chains and use their delegated ATOMs to earn rewards on the child chains as well as risk their parent chain ATOMs to slashable events should they misbehave on child chains.
```

Comment:
Should they still undelegate if they are validating 0 child chains

Reply:
I think no, it will take time to accumulate enough delegators to operate in different capacities—on the cosmos hub itself, or as a potential validator on a child chain. If a validator is able to get delegations without yet producing blocks I think it would be a testament to their potential rather than a technical mistake that should be corrected. Certainly if they never produce blocks on any networks the delegators should consider redelegating.